src/igvfd/schemas/image_file.json

```diff
@@ -0,0 +1,86 @@
+{
+    "title": "Image File",
+    "$id": "/profiles/image_file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "A file containing image data in jpg or png format.",
```

Comment:
I think the description should not limit to specific file formats.

Reply:
fixed.